_P208_Nokia_LCD_5110.ino

```diff
@@ -217,32 +224,45 @@ boolean Plugin_208(byte function, struct EventStruct *event, String& string){
             argIndex = string.lastIndexOf(',');
             tmpString = string.substring(argIndex + 1);
             if (tmpString.equalsIgnoreCase(F("Clear"))){
-              addLog(LOG_LEVEL_INFO, F("Clear Display"));
+              myLog("Clear Display");
               lcd3->clearDisplay();
               lcd3->display();
             }
             if (tmpString.equalsIgnoreCase(F("blOn"))){
               success = true;
               PCONFIG(0) = 1;
-              setBacklight(event); 
+              setBacklight(event);
             }
             if (tmpString.equalsIgnoreCase(F("blOff"))){
               success = true;
               PCONFIG(0) = 0;
-              setBacklight(event); 
+              setBacklight(event);
             }
-          break;
-        }
+            break;
+          }
         }
       }
   }  // switch (function)
   return success;
 }
 
+//void myLog (String message) {
+//  addLog(LOG_LEVEL_INFO, "P208: " + message);
+//}
+void myLog(const String & message) {
+  String log;
+  if (sentlog){
+    log.reserve(message.length() + 6);
+    log += F("P208: ");
+    log += message;
+    addLog(LOG_LEVEL_INFO, log);
+  }
+}
+
 void setBacklight(struct EventStruct *event) {

```

Comment:
Same here, please prefix the function name

Reply:
done